executor/distsql.go

```diff
@@ -55,6 +55,22 @@ func resultRowToRow(t table.Table, h int64, data []types.Datum, tableAsName *mod
 	return &Row{Data: data, RowKeys: []*RowKeyEntry{entry}}
 }
 
+// calculateGeneratedColumns calculates all virtual generated columns, with such arguments:
+//  cols is the collection of columns of vs;
+//  gvs's keys are index in cols, and the values are generation expressions.
```

Comment:
There are two term: `generated column` and `generation expression`, just as same with MySQL.